swarm/api/http/server.go

```diff
@@ -1011,6 +1012,8 @@ func (b bufferedReadSeeker) Seek(offset int64, whence int) (int64, error) {
 }
 
 func (s *Server) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
+	ctx := context.TODO()
```

Comment:
I think that this ctx should not be created, as the context propagation for http is done with Request.